tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
Hello everyone,

I’m wondering about the plans for the meme and fimo tools currently contained within the Galaxy code base.  I think I’ve seen these tools on Galaxy main in the recent past, but searching for them now does not find them, so perhaps I’m wrong.  

Anyway, these 2 tools are needed for the ChIP-exo Galaxy flaovr, so I’m wondering if they can be moved out of the Galaxy core and into the tools-iuc repository.  It also loooks like they need to be cleaned up a bit.  I would be happy to do this, please let me know how I should go about it.

Thanks!

Greg Von Kuster